### DISCUSSION TITLE
Assembly token gone from Firefly Wallet since weeks

### DISCUSSION THREAD

**Discussion by Patrick-vs** [iotaledger/firefly#7683] [Help & Support]
Hello,

Few weeks ago I recognised that my Assembly token are no more visible within the firefly wallet. My shimmer token which I sent to the shimmer wallet are still visible. Or are the Assembly token changed automatically to MIOTA Tokens? Cause I have a few more IOTA tokens now in my wallet than before. But I cannot see the swap for that within the transaction history. Thank you very much for help!

**Comment by begonaalvarezd** [comment]
Hi, the network got recently upgraded so the old firefly (v1.7.x) doesnt work anymore, you need to update to the latest, which as of today is 2.0.3. If you use ledger please update your iota ledger app to v0.8.6.

Quick information about ASMB: please read https://blog.iota.org/stardust-upgrade-iota-tokenomics/. The assembly project does not exist anymore, but your rewards got airdropped to you in form of IOTAs. They will appear in the new version 2.0.3 in the vesting tab. If you are missing balances you can run the balance finder from settings > advanced > balance finder, or if you are missing rewards you can also run the rewards finder from the vesting tab, the '....' menu, search vesting rewards

Will close the issue because the new firefly app is not intended to display old staking rewards, but the new iota airdrop is on the new app. 

Please let us know if you have any other issues

**Reply by Patrick-vs** [reply]
Hello , ok thank you for the detailed explanation. Now it is clear that my Assembly token are not gone and will swap step by step into MIOTA token. It is awesome too

**Comment by Patrick-vs** [comment]
Hello again,

Today I received a NFT drop were I can gain 2000 IOTA on the www.iotalottery.com website, but this website doesn't exist. Is this a scam? I did already claim the NFT accidentally but did not enter any address or sth. Hopefully my Firefly Wallet is still save. 

Thank you very much in advance for your help!!

Regards
Pat

**Reply by begonaalvarezd** [reply]
Its a scam, you can burn the nft from the nft details > '...' menu > burn
You wallet is safe as soon as you dont enter your mnemonic to any random website like this one

**Comment by Patrick-vs** [comment]
Oh ok, many thanks for the fast reply and advice . I did burn it successfully !!!

**Comment by elipsion-dev** [comment]
I hadn't opened Firefly in a year or so... it showed a balance of over $2,000 worth of Assembly tokens. Then I updated and it now shows a balance of $6 in IOTA... what the fuuuuuuuuuuuuu

**Reply by begonaalvarezd** [reply]
Please read and follow these steps https://github.com/iotaledger/firefly/discussions/7683#discussioncomment-7466939
Let us know if the issue persists

**Reply by elipsion-dev** [reply]
I tried both of the things from that comment. I searched for rewards in vesting and I searched from the advanced menu in settings. It didn’t find anything else. 
![IMG_4702](https://github.com/iotaledger/firefly/assets/72885319/0ae0b92d-a08f-4b51-bdea-27eb6001036a)
![IMG_4712](https://github.com/iotaledger/firefly/assets/72885319/42c93020-02a2-4e02-9c0b-68549df08b8a)

**Reply by begonaalvarezd** [reply]
As mentioned in the comment above "the ASMB project does not exist anymore, so you will not see ASMB tokens in your wallet, instead you must have gotten an IOTA airdrop that you can see from the vesting tab, if you cant see any IOTAs in the vesting tab, I suggest you keep running the balance finder within the vesting tab or settings (the one in the vesting tab is faster if you know with which wallet you staked, wallet 1, wallet 2, etc..)
> Quick information about ASMB: please read https://blog.iota.org/stardust-upgrade-iota-tokenomics/. The assembly project does not exist anymore, but your rewards got airdropped to you in form of IOTAs. They will appear in the new version in the vesting tab

I recommend you read the post linked about the tokenomics upgrade, the new tokenomics are explained there

**Reply by elipsion-dev** [reply]
@begonaalvarezd I have followed your instructions, read the documents tried all of these methods and still my $2k is now $6. Please help or suggest to me somewhere I can get help. I’ve never made a single transaction aside from staking, and I can show my account activity and proof that my $2k is now only $6. Please advise. Thank you

**Comment by elipsion-dev** [comment]
Yea I realize Assembly is done and that value is supposed to get airdropped
in the form of iota tokens. Despite running the balance searches probably
50 times each, the over $2,000 worth of Assembly I had turned into $6 of
iota. The entire value of all my future airdrops is $14. So is that $2k
just gone because the iota tokens I was given in their place aren’t worth
the same? I’ve never been part of a project where $2,000 just disappeared
because of an upgrade or update.
